Generator/DoctrineGenerator.php

```diff
@@ -102,52 +95,4 @@ protected function doBuild()
             )
         );
     }
-
```

Comment:
Actually, I see a difference compared to before. In current implementation, for embed types to work, Edit and New Controller need to be created (whereas they are not necessary).I will so bring back embedTypes.